.github/workflows/unit-tests.yaml

```diff
@@ -12,6 +12,45 @@ on:
     - reopened
     - synchronize
 jobs:
+  go-postgres:
+    strategy:
+      matrix:
+        gotags: [ 'GOTAGS=""', 'GOTAGS=release' ]
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.49
+#    services:
+#      postgres:
+#        image: quay.io/stackrox-io/central-db:3.73.x-369-gee573ecd8d #TODO: Change to latest or current tag
+#        env:
+#          PGDATA: /tmp/data
+#          POSTGRES_PASSWORD: postgres
+#        options: >-
+#          --entrypoint docker-entrypoint.sh
+#          --health-cmd pg_isready
+#          --health-interval 10s
+#          --health-timeout 5s
+#          --health-retries 5
```

Comment:
I'd suggest to remove this dead code. If you'd like to resurrect the `service` approach some day, maybe you can add a TODO comment instead.
```suggestion
```